packages/storage-documents/src/privacy-scan-types/privacy-page-scan-request.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { ItemType } from '../item-type';
+import { StorageDocument } from '../storage-document';
+import { WebsiteRequest, ReportGroupRequest } from '../on-demand-page-scan-batch-request';
+
+export interface PrivacyPageScanRequest extends StorageDocument {
```

Comment:
no need for this type, we will use existing structure with `privacyScan?: boolean;` flag